.gitignore

```diff
@@ -1,4 +1,7 @@
+package-lock.json
```

Comment:
We should commit the lock.

Reply:
aren't locks just for applications? this is a library, so the versions should match what the end user uses.